README.md

```diff
@@ -5,10 +5,39 @@
 
 Canonically pronouced *nice*
 
-A compositional statistics and visualization toolbox
 
-You can install Gneiss via pip
+ compositional statistics and visualization toolbox
++gneiss is a compositional statistics and visualization toolbox.  
+ 
+Note that gneiss is not compatible with python 2, and is compatible with Python 3.4 or later.
+gneiss is currently in alpha.  We are actively developing it, and __backward-incompatible interface changes can and will arise__.
```

Comment:
At the moment, no.  This is more of a defensive statement.

But I can see this happening.  For instance, we could change the file structure of the repository to clean up the code.